masterlist.yaml

```diff
@@ -136,3 +136,63 @@ plugins:
     url: [ 'http://www.nexusmods.com/fallout4/mods/2142' ]
     after:
       - 'Simple Bug Fixes.esp'
+
+  - name: 'DarkerNights-TrueStorms-LITE-Audio.esp'
+    url: [ 'http://www.nexusmods.com/fallout4/mods/191' ]
+    after:
+      # Ensure the magic of Darker Nights is applied to WTHR records modified by the True Storms LITE Audio addon
+      - 'TrueStormsFO4-LITE-Audio.esp'
+    msg:
+      - type: warn
+        content: 'This plugin does not match the active True Storms plugins. Reinstall the **Darker Nights for True Storms** patch and select the option which matches your True Storms installation.'
+        lang: eng
+        condition: 'not(active("TrueStormsFO4-LITE-Audio.esp")) and active("DarkerNights-TrueStorms-LITE-Audio.esp")'
+    inc:
+      - 'DarkerNights-TrueStorms.esp'
+
+  - name: 'DarkerNights-TrueStorms.esp'
+    url: [ 'http://www.nexusmods.com/fallout4/mods/191' ]
+    msg:
+      - type: warn
+        content: 'This plugin does not match the active True Storms plugins. Reinstall the **Darker Nights for True Storms** patch and select the option which matches your True Storms installation.'
+        lang: eng
+        condition: 'active("TrueStormsFO4-LITE-Audio.esp") and active("DarkerNights-TrueStorms.esp")'
+    inc:
+      - 'DarkerNights-TrueStorms-LITE-Audio.esp'
+
+  - name: 'TrueStormsFO4.esp'
+    url: [ 'http://www.nexusmods.com/fallout4/mods/4472' ]
+    after:
+      - 'DarkerNights.esp'
+    msg:
+      - type: say
+        content: 'This plugin conflicts with plugin ''DarkerNights.esp''. Install and activate the appropriate Darker Nights patch.'
```

Comment:
I wonder if possible whether messages like this should be made more generic and added to commons as something like:

`This plugin conflicts with plugin '%1%', however a patch is available.`

My wording is terrible, but you get the idea.